test/verifiedTests/analysis/testMultiSpeciesModelling/testMgPipe.m

```diff
@@ -0,0 +1,119 @@
+% The COBRAToolbox: testMgPipe.m
+%
+% Purpose:
+%     - test the mappingInfo function of the Microbiome Toolbox
+%
+% Authors:
+%     - original version: Federico Baldini & Laurent Heirendt, April 2018
+%
+
+global CBTDIR
+
+% define the features required to run the test
+requiredToolboxes = { 'distrib_computing_toolbox' };
+
```

Comment:
Isn't CPLEX required? 
And you are not testing for the requirements.
Is that intentional?

Reply:
Yes - the solver functions will only be tested later.

Reply:
This test fails if the parallel toolbox is not installed instead of being skipped.